packages/rest/index.ts

```diff
@@ -11,6 +11,7 @@
 
 export * from "./src/client/types/HTTPVerb";
 export * from "./src/client/doc/IHeaderContent";
+export * from "./src/client/doc/IRestOptions"
```

Comment:
Hi @apsychogirl . Thanks for taking the time to contribute these fixes.
I can see that the build is failing with linting error on this line - `missing semicolon`. Could you correct it, please?

Reply:
Sure!